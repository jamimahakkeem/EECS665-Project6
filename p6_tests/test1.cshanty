void main() {
	int a;
	a = -1;
	a++;
	a--;
	report a;
	receive a;
}