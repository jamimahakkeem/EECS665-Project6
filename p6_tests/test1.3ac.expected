[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN main LOCALS]
a (local var of 8 bytes)
tmp0 (tmp var of 8 bytes)
[END main LOCALS]
main:       enter main
            [tmp0] := NEG64 1
            [a] := [tmp0]
            [a] := [a] ADD64 1
            [a] := [a] SUB64 1
            REPORT [a]
            RECEIVE [a]
lbl_0:      leave main
